scripts/conflicts/diff.js

```diff
@@ -20,37 +26,90 @@ if (!server || argv.h || argv.help) {
 
 console.log(`Generating diffs for conflicts on ${server}`);
 
+
 const DB = new PouchDB(server);
 
-DB.query('medic-conflicts/conflicts')
+
+const writeToFile = function(path, content) {
+  const writeStream = fs.createWriteStream(path);
```

Comment:
You don't need to use `createWriteStream` here. It's simpler if you just use `writeFile` or `writeFileSync`: 
```suggestion
fs.writeFileSync(path, JSON.stringify(content, null, 2));
```

Another note: we use 2 spaces for indentation, so it's helpful to have the same format across your json as well.